src/main/java/dal/fatih/todoly/Main.java

```diff
@@ -43,17 +68,19 @@ public static Task handleCreateTask() {
     }
 
     public static void listAllTasks() {
-        if (tasks.isEmpty()) {
-            System.out.println("Task list is empty");
-        } else {
-            for (Map.Entry task : tasks.entrySet()) {
-                System.out.println(task.getValue());
-                System.out.println("----------------------------------");
-            }
+
+        Set set = tasks.entrySet();
+        Iterator iterator = set.iterator();
+        while (iterator.hasNext()) {
+            Map.Entry task = (Map.Entry) iterator.next();
+
+            System.out.println(task.getValue());
+            System.out.println("----------------------------------");

```

Comment:
Why changing the way it was? I think the previous version was looking good already. Make sure there is no change with this function.